pac4j-oauth/pom.xml

```diff
@@ -27,15 +27,16 @@
 	<packaging>jar</packaging>
 	<name>pac4j for OAuth protocol</name>
 
+	<properties>
+		<scribe.version>2.2.2</scribe.version>
+		<jackson.version>2.7.1-1</jackson.version>
```

Comment:
You likely should stay with 2.6 on jackson. 2.7 is not supported by Spring yet, and I imagine that might break a lot of existing uses.